### PR TITLE
Add streaming tests

### PR DESCRIPTION
**Pull request by michaelbausor**
Note that this PR is against a branch because of testing/merging issues

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
@shinfan could you take a look first?

File: generated/php/google-cloud-speech-v1/tests/unit/Speech/V1/SpeechClientTest.php [hunk 1]
```diff
@@ -301,4 +304,89 @@ public function longRunningRecognizeExceptionTest()
...
+        $this->assertNull($streamRequestObject);
+
+        $callObjects = $grpcStub->getCallObjects();
+        $this->assertSame(1, count($callObjects));
+        $bidiCall = $callObjects[0];
+
+        $writeRequests = $bidiCall->getReceivedCalls();
+        $this->assertSame(3, count($writeRequests));
+        foreach ($writeRequests as $writeRequest) {
+        }
```
Comment (shinfan) [inline generated/php/google-cloud-speech-v1/tests/unit/Speech/V1/SpeechClientTest.php]: I think we can get rid of this foreach block or is it something going wrong here?

File: generated/php/google-cloud-speech-v1beta1/tests/unit/Speech/V1beta1/SpeechClientTest.php [hunk 2]
```diff
@@ -293,4 +296,91 @@ public function asyncRecognizeExceptionTest()
...
+        $this->assertSame('/google.cloud.speech.v1beta1.Speech/StreamingRecognize', $streamFuncCall);
+        $this->assertNull($streamRequestObject);
+
+        $callObjects = $grpcStub->getCallObjects();
+        $this->assertSame(1, count($callObjects));
+        $bidiCall = $callObjects[0];
+
+        $writeRequests = $bidiCall->getReceivedCalls();
+        $this->assertSame(3, count($writeRequests));
+        foreach ($writeRequests as $writeRequest) {
```
Comment (shinfan) [inline generated/php/google-cloud-speech-v1beta1/tests/unit/Speech/V1beta1/SpeechClientTest.php]: Same here

File: generated/php/google-cloud-speech-v1beta1/tests/unit/Speech/V1beta1/SpeechClientTest.php [hunk 3]
```diff
@@ -293,4 +296,91 @@ public function asyncRecognizeExceptionTest()
...
+        $bidi->write($request);
+        $responses = [];
+        $responses[] = $bidi->read();
+
+        $bidi->writeAll([$request, $request]);
+        foreach ($bidi->closeWriteAndReadAll() as $response) {
+            $responses[] = $response;
+        }
+
+        $this->assertEquals([$expectedResponse, $expectedResponse, $expectedResponse], $responses);
```
Comment (shinfan) [inline generated/php/google-cloud-speech-v1beta1/tests/unit/Speech/V1beta1/SpeechClientTest.php]: It is not that useful to compare those responses as they are all identical. (For example, if somehow the gapic layer messes around the ordering of the response, we would not be able to catch it here)

**Comment by michaelbausor** [comment]
Replacing this PR with https://github.com/googleapis/api-client-staging/pull/341 now that testing/merging issues are resolved

**Pull request closed without merging**